desktop-exporter/server.go

```diff
@@ -18,13 +18,13 @@ func getTracesHandler(traceStore *TraceStore) func(http.ResponseWriter, *http.Re
 		traceStore.mut.Lock()
 		defer traceStore.mut.Unlock()
 
-		writer.WriteHeader(http.StatusOK)
-
-		traces, err := json.Marshal(traceStore.traceMap)
+		jsonTraces, err := json.Marshal(traceStore.traceMap)
 		if err != nil {
-			fmt.Printf("error marshalling traceStore: %s\n", err)
+			panic(fmt.Errorf("error marshalling traceStore: %s\n", err))
```

Comment:
This probably doesn't merit a `panic` but we can iterate on our endpoints later.